crates/bevy_ui/src/ui_node.rs

```diff
@@ -398,6 +434,40 @@ impl Default for AlignItems {
     }
 }
 
+/// How items are aligned according to the cross axis
+#[derive(Copy, Clone, PartialEq, Eq, Debug, Serialize, Deserialize, Reflect)]
```

Comment:
```suggestion
#[derive(Copy, Clone, PartialEq, Eq, Debug, Serialize, Deserialize, Reflect, ReflectFrom)]
```

I think the `ReflectFrom` derive is all that is needed for options to work.

Reply:
I think that should be `FromReflect` :)

Reply:
can't manage a single line nowadays without autocomplete 😅

Reply:
Hmm... What about `Vec`? That's actually more critical: `Option` which has a reasonable workaround, but I absolutely need to have `Vec<GridTrack>`

Reply:
iirc `FromReflect` makes it work for `Vec` as well

Reply:
Ah yes, that does seem to be working now. I think Rust Analyzer got confused when I first tried that, which made me think it wasn't working.